File: spring-cloud-bus-demo/bus-common/src/main/java/com/livk/bus/listener/EnableRemoteEventListener.java
package com.livk.bus.listener;

import org.springframework.context.annotation.Import;

import java.lang.annotation.*;

/**
 * <p>
 * EnableRemoteEventListener
 * </p>
 *
 * @author livk
 * @date 2022/1/7
 */
@Target({ ElementType.TYPE })
@Retention(RetentionPolicy.RUNTIME)
@Documented
@Import(LivkEventListener.class)
public @interface EnableRemoteEventListener {

}

File: spring-cloud-bus/bus-commons/src/main/kotlin/com/livk/bus/listener/EnableRemoteEventListener.kt
package com.livk.bus.listener

import org.springframework.context.annotation.Import

/**
 * @author livk
 */
@Target(AnnotationTarget.CLASS)
@Retention(AnnotationRetention.RUNTIME)
@MustBeDocumented
@Import(
    LivkEventListener::class
)
annotation class EnableRemoteEventListener
